Kaggle/Titanic/titanic_analysis3.ipynb

```diff
@@ -34,11 +34,11 @@
    "execution_count": 1,
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        sns.countplot(x=catColumns[i], data=df, hue=targetColumn, ax=axs[currCol])
</pre><p>how about this countplot eh?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ronald-leung/datascience_machinelearning/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>